vcpkg/get_started/get-started-linux.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Install and use packages on Linux with vcpkg
+description: Tutorial guides the user through the process of installing and using packages on Linux with vcpkg.
+author: JavierMatosD
+ms.author: javiermat
+ms.topic: tutorial
+ms.date: 08/23/2023
+ms.prod: vcpkg
+#CustomerIntent: As a beginner C++ developer, I want to learn how to install and manage packages on a Linux environment using vcpkg, so that I can easily set up and maintain C++ projects with necessary dependencies.
+---
+
+# Tutorial: Install and use packages on Linux with vcpkg
+
+This tutorial shows how to set up a C++ "Hello World" CMake project using the `fmt` library. Manage dependencies, construct the application, and execute the project with the help of vcpkg. This guide is tailored for a Linux environment.
+
+## Prerequisites
+
+- A terminal to work in.
+- `git`, `CMake`, and a C++ compiler installed on your system
+- vcpkg installed on your system
+
+## 1 - Set up the project
+
+1. Create the project directory
+
+    This tutorial assumes your project is located in `/home/helloworld`. If you intend to place your project somewhere else, replace any references to that path with your preferred path.
```

Comment:
```suggestion
    This tutorial assumes your project is located in `/src/helloworld`. If you intend to place your project somewhere else, replace any references to that path with your preferred path.
```

`/home` is an awkward assumption since it's a significant folder on Linux. Alternative suggestions: `/src/hello`, `~/hello`, `$HOME/hello`, `/opt/src/hello`, `/home/user/hello`

Reply:
Personally, I'd go with `~/hello`.